worker/src/RTC/RtpDictionaries/RtpCodecMimeType.cpp

```diff
@@ -35,6 +35,7 @@ namespace RTC
 		// Video codecs:
 		{ "vp8",             RtpCodecMimeType::Subtype::VP8             },
 		{ "vp9",             RtpCodecMimeType::Subtype::VP9             },
+		{ "av1",             RtpCodecMimeType::Subtype::AV1             },
```

Comment:
Can we add it after h265 for consistency with `supportedRtpCapabilities.ts` and because this is the last added codec?